src/stacker/ConcreteStacker.ts

```diff
@@ -25,6 +25,27 @@ export class ConcreteStacker implements Stacker {
     this.repositoryUpdateListener(repo);
   };
 
+  private localRefPrefix : string = "refs/heads/";
+
+  private refIsLocal(refString: string) : boolean {
+    return refString.startsWith(this.localRefPrefix);
+  }
+
+  private localRefToBranchName(refString: string) : string {
+    if (!this.refIsLocal(refString)) return refString;
+    return refString.substr("refs/heads/".length);
```

Comment:
I know I wrote the original code in GitLocal but let me nitpick my own code 😆 
```suggestion
    return refString.substr(this.localRefPrefix.length);
```